Formula/schemathesis.rb

```diff
@@ -0,0 +1,177 @@
+class Schemathesis < Formula
+  include Language::Python::Virtualenv
+
+  desc "Testing tool for web applications with specs"
+  homepage "https://schemathesis.readthedocs.io/"
+  url "https://files.pythonhosted.org/packages/0e/81/f9198ed48e4370278f49889a6ed2f92effdbc9d0f1e538a62f1e55a80fde/schemathesis-3.16.1.tar.gz"
+  sha256 "e2480aa4aacf4053519170c1ccdfd1347ea30a303b7f76ce43c2155073c677e2"
+  license "MIT"
+
+  depends_on "python@3.10"
+  on_linux do
+    depends_on "rust" => :build  # for cryptography
```

Comment:
There doesn't appear to be a `cryptography` resource?

Reply:
Yeah, it's weird. One or more of these resources (at least [`hypothesis_graphql`](https://github.com/Stranger6667/hypothesis-graphql), possibly others as well) declares its dependencies using [poetry](https://python-poetry.org) in a `pyproject.toml` file. When Homebrew tries to install this resource using `pip`, `pip` ends up installing poetry in order to do the installation. One of poetry's dependencies is `cryptography`, and `cryptography` needs Rust to be installed.

In earlier commits of this pull request, I tried adding `depends_on "poetry"` to prevent `pip` from trying to install it, but that doesn't seem to make any difference. It's a strange behavior, and it means that we are downloading files in the install step rather than beforehand, but I can't find a way to avoid it. Do you have any suggestions for how to fix this?

Reply:
I also don't understand why installing Rust is only necessary on Linux. The CI servers running macOS pass just fine without installing Rust first.

Reply:
There are a few formulae that do `depends_on "poetry"` in order to avoid `pip` installing it itself and requiring `rust`. Try having a look at those for examples of what to do here.

Reply:
Since https://github.com/Stranger6667/hypothesis-graphql/pull/83 was merged + released, we shouldn't even need `poetry` anymore after updating the resources